Road/source/data/get_data.py

```diff
@@ -0,0 +1,21 @@
+from telethon import TelegramClient,events
+import csv
+
+api_id = 28394694 
+api_hash = '4c0b02c61f2626bd3f0d1b58883ec759'
+client = TelegramClient('session_name', api_id, api_hash) # create a client object to connect to telegram using my account info
+client.start()
+
+@client.on(events.NewMessage(chats='AhwalTareq')) # create event handler to listen to AhwalTareq for new messages
+async def my_event_handler(event)->None:
+    """ this function get called by the event handler 
+    when we recive new message from AhwalTareq group
+
+    Args:
+        event : the payload from telegram server contain the information about the new message
+    """
+    with open('data.csv', 'a+', newline='',encoding="utf-8") as csvfile: 
```

Comment:
and make sure there are spaces between variables. (newline = "", encoding ..) (import TelegramClient, events), etc
you can use a pep8 extension for this.

Reply:
pep8 extension doesn't work on my machine I don't know why, so I don't follow every space and comma in the convention because its a waste of time doing it manually 🤢 , I will tell @mawada-sweis to reformate them after the merge